packages/stt-adapters/digital-paper-edit/index.js

```diff
@@ -3,74 +3,68 @@
  * More details see
  * https://github.com/bbc/digital-paper-edit
  */
-import generateEntitiesRanges from '../generate-entities-ranges/index.js';
-import groupWordsInParagraphsBySpeakers from './group-words-by-speakers.js';
+import generateEntitiesRanges from '../generate-entities-ranges';
+import groupWordsInParagraphsBySpeakers from './group-words-by-speakers';
 /**
  * groups words list from kaldi transcript based on punctuation.
  * @todo To be more accurate, should introduce an honorifics library to do the splitting of the words.
  * @param {array} words - array of words opbjects from kaldi transcript
  */
-const groupWordsInParagraphs = words => {
+const groupWordsInParagraphs = (words) => {
   const results = [];
   let paragraph = { words: [], text: [] };
 
-  words.forEach(word => {
+  words.forEach((word) => {
+    paragraph.words.push(word);
+    paragraph.text.push(word.text);
+
     // if word contains punctuation
     if (/[.?!]/.test(word.text)) {
-      paragraph.words.push(word);
-      paragraph.text.push(word.text);
       paragraph.text = paragraph.text.join(' ');
       results.push(paragraph);
       // reset paragraph
       paragraph = { words: [], text: [] };
```

Comment:
Since you have to reset this each time, could you just define `paragraph` for the first time inside the `forEach` loop?

Reply:
good point